.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches:
      - main
      - master
  pull_request:
```
Yeah this will prevent running jobs 2x on PRs

---

how does it doe this?